scripts/pi-hole/php/FTL.php

```diff
@@ -35,9 +35,16 @@ function getResponseFTL()
 
 	$response = [];
 
+	$errCount = 0;
 	while(true)
 	{
 		$out = fgets($socket);
+		file_put_contents('/tmp/pihole.txt', $out, FILE_APPEND);
```

Comment:
Why do you want to write to this file?

Reply:
Sorry, that was a debug output that I just removed.